src/chrome/content/rules/CNBC.com-mixed.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+	For more CNBC.com related, see CNBC.com.xml
+-->
+<ruleset name="CNBC.com (mixed)">
```

Comment:
This rule will break `webcast.cnbc.com`. You should either use the mixedcontent platform or exclusions.